workstreams/events_in_cicd/vocabulary.md

```diff
@@ -0,0 +1,108 @@
+# CD Events Vocabulary Proposal
```

Comment:
Really good work! I added a bunch of comments, but they are all minor.

Reply:
@erkist thanks a lot for the comments.. let's see if we get more feedback.. but we definitely need to discuss this PR in our next meeting.. it might be good to highlight some of your comments and share them with the group. 

@afrittoli if you have time to take a look at these definitions, it will be great to have your feedback on this.